src/main/java/com/lpnu/shaggybeavers/facade/RelicFacade.java

```diff
@@ -13,4 +17,16 @@ public class RelicFacade {
 
     private final RelicService relicService;
 
+    private final StorageFacade storageFacade;
+
+    public URL uploadFile(MultipartFile multipartFile, Long relicId) throws IOException {
+        URL url = storageFacade.uploadFile(multipartFile);
+        relicService.setImageUrl(url, relicId);
+        return url;
+    }
+
+    public URL downloadFile(String fileName) {
+        return storageFacade.downloadFile(fileName);
```

Comment:
you can save url in the relic model and take that from model and return or you can store file name and take it from s3 bucket but there you did it wrong